src/dbnode/storage/index.go

```diff
@@ -905,18 +1138,19 @@ func (i *nsIndex) CleanupExpiredFileSets(t time.Time) error {
 
 func (i *nsIndex) Close() error {
```

Comment:
is there not a "Tick()" loop? How does the ticking get stopped

Reply:
There is a tick loop yeah, see `Tick()` on `nsIndex`.